src/components/informationCard/InformationCard.css

```diff
@@ -0,0 +1,32 @@
+.infoCard {
+    flex: 0 1 calc(50% - 30px);
+    background-color: var(--color-background-light);
+    border-radius: 20px;
+    border: none;
+    padding: 30px;
+    position: relative;
+}
+
+.info-card-title {
+    font-size: 35px;
+    font-weight: 400;
+}
+
+.info-card-content {
+    line-height: 2.5;
+}
+
+.info-card-experience-tag {
```

Comment:
Hier hebben we twee keer dezelfde CSS! Kun je eens kijken hoe je dezelfde declaratie kunt gebruiken voor twee selectors, zonder de declaratie te herhalen?

Reply:
Met een komma, list combinator.